contracts/coordinator/src/contract.rs

```diff
@@ -57,11 +57,8 @@ pub fn execute(
             execute::check_governance(&deps, info)?;
             execute::register_prover(deps, chain_name, new_prover_addr)
         }
-        ExecuteMsg::SetActiveVerifiers { next_verifier_set } => {
-            execute::set_active_verifier_set(deps, info, next_verifier_set)
-        }
-        ExecuteMsg::SetNextVerifiers { next_verifier_set } => {
-            execute::set_next_verifier_set(deps, info, next_verifier_set)
+        ExecuteMsg::SetVerifiers { verifiers } => {
```

Comment:
This should be more descriptive, just looking at the interface I would have no idea what this is for. I'm fine still calling it `SetActiveVerifiers` (ask the rest of the team about it) or maybe `SetVerifiersInRotation`?

Reply:
`SetActiveVerifiers` is fine with me